guides/source/error_reporting.md

```diff
@@ -158,3 +173,19 @@ module MySdk
   end
 end
 ```
+
+When using the Rails error reporter, errors may be reported multiple times. For instance, calling `Rails.error.record` will report the error via your subscriber, then raise it. If you have other error mechanisms like a Rack middleware, they will then capture the exception and report it again, then raise it once more, where it may go through your subscriber a second time. 
```

Comment:
@casperisfine this was something I noticed lately. Since the executor calls `Rails.error.report`, if I call `Rails.error.record` in my controller, the error will be reported twice? Do you think this is worth fixing?  FWIW, the fix would probably be the approach I described below of adding an instance variable, but doing it in the executor. I can send in a PR for that.

Reply:
> if I call `Rails.error.record` in my controller, the error will be reported twice?

Why would you do that though? The point of `record` is when you know the exception will be handled by the caller at some point.

Reply:
The user may add more context in the controller's `Rails.error.record`, so the error captured there will have more context than when being captured by the executor. Sentry also has a similar mechanism to workaround the double-reporting issue.

Reply:
> The user may add more context in the controller's Rails.error.record

Interesting, I didn't think of that. I'll think a bit more about it, but yeah, it may make sense to dedup in that case.